### DISCUSSION TITLE
Music controls on Linux system tray.

### DISCUSSION THREAD

**Discussion by cakePhone** [ciderapp/Cider#1018] [Rejected Features]
So, I've recently been finding myself in an awkward situation here, I switched from windows 11 to Ubuntu 22.04 and I realized there's no native music player controls on the system tray with Cider. I really think this would be a great addition and pretty much a great quality of life improvement as I don't want to open the app each time just to pause a song or skip it.

**Comment by cryptofyre** [comment]
@coredev-uk

**Reply by coredev-uk** [reply]
no thanks

**Comment by coredev-uk** [comment]
As Cider does have MPRIS support, you can control the playback within the app using the [mpris-remote](http://manpages.ubuntu.com/manpages/bionic/man1/mpris-remote.1.html) command in Ubuntu 18.04 or later.
